include/EVT/io/platform/f3xx/f302x8/CANf302x8.hpp

```diff
@@ -44,6 +44,16 @@ class CANf302x8 : public CAN {
      */
     CANMessage* receive(CANMessage* message, bool blocking = false);
 
+    /**
+     * @copydoc EVT::core::IO::CAN::addCANFilter
```

Comment:
The copy docs have been removed since the docstrings are automatically inherited, this should already be the case in `main` so you may just need to merge.